count_words.py

```diff
@@ -29,5 +29,5 @@ def print_summary(filepath, ct):
         ', '.join(counts)))
 
 
```

Comment:
can you pls make this words_list to word_list?

Reply:
Actually we dont need to change this